internal/generate/olm-catalog/csv.go

```diff
@@ -44,13 +44,20 @@ import (
 )
 
 const (
-	olmCatalogChildDir = "olm-catalog"
-	OLMCatalogDir      = scaffold.DeployDir + string(filepath.Separator) + olmCatalogChildDir
-	CSVYamlFileExt     = ".clusterserviceversion.yaml"
+	OLMCatalogChildDir = "olm-catalog"
+	// OLMCatalogDir is the default location for OLM catalog directory.
+	OLMCatalogDir  = scaffold.DeployDir + string(filepath.Separator) + OLMCatalogChildDir
+	csvYamlFileExt = ".clusterserviceversion.yaml"
 
-	BundleDirKey = "bundle"
+	// Input keys for CSV generator whose values are the filepaths for the respective input directories
+
+	// DEBUG: Why is this an input key? Users don't need to configure this.
```

Comment:
@estroz

Reply:
Figured this out as I'm writing the test for this. 
`BundleDirKey` is just being used to check for a preexisting CSV bundle directory so we can update from that when generating the CSV.

Going to refactor `BundleDirKey ` to not be a generator key and just be a private field of the  `csvGenerator` since it's something internal and not meant to be an input.